packages/govern-create/helpers/logger.ts

```diff
@@ -0,0 +1,46 @@
+import chalk, { ForegroundColor } from 'chalk'
+
+const infoTag = chalk.blue('[INFO] ')
+const errorTag = chalk.red('[ERROR] ')
+const warnTag = chalk.yellow('[WARN] ')
+
+export function _prependTag(
+  lines: string,
+  tag: string,
+  color?: typeof ForegroundColor
+): string {
+  if (color) tag = chalk[color](tag)
+  return lines
+    .split('\n')
+    .map((line) => tag + line)
+    .join('\n')
+}
+
+export function logInfo(data: string): void {
```

Comment:
I would create a simple static class for such a logger and use log levels you can pass instead of exposing different functions. This will definitely lower the code duplication here.